tests/test_infer_subminor_version.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euxo pipefail
+
+readonly CURRENT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)
+readonly REPO_DIR=$(realpath "${CURRENT_DIR}/..")
+
+readonly TEST_SUFFIX=$(tr -dc 'a-z0-9' </dev/urandom | head -c 6)
+readonly TEST_BUCKET="gs://dataproc-custom-images-presubmit"
+readonly TEST_ZONE="us-central1-a"
+
+test_inferred_subminor_version_debian() {
```

Comment:
Could you add a verification for the result image, check the `goog-dataproc-version` label is set with the resolved subminor version?

Reply:
Seems like we do not this for other tests, so let's tackle this separately.